src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -129,7 +129,16 @@ public static boolean equals(PersonName personName, String string) {
             return nullValuesCheck;
         }
 
-        return personName.toString().equals(string);
+        return StringUtils.equalsIgnoreCase(normalise(personName.toString()),normalise(string));
+    }
+
+    public static boolean equals(String corporateName, String string) {
```

Comment:
This will make all string comparisons ignore case. Should this be named something like: ComparisonHelper.equalsIgnoreCase()